Documentation/auto-dependency-flow/api.md

```diff
@@ -141,22 +141,20 @@ Use the AssetRoot properties if they are defined. Example (adapted from [CLI's b
 
   <PropertyGroup>
     <CoreSetupBlobRootUrl Condition="'$(DotNetAssetRootUrl)' != ''">$(DotNetAssetRootUrl)</CoreSetupBlobRootUrl>
-    <CoreSetupBlobRootUrl Condition="'$(CoreSetupBlobRootUrl)' == ''">https://dotnetcli.azureedge.net/dotnet/</CoreSetupBlobRootUrl>
-
-    <CoreSetupRootUrl>$(CoreSetupBlobRootUrl)Runtime/</CoreSetupRootUrl>
+    <CoreSetupBlobRootUrl Condition="'$(CoreSetupBlobRootUrl)' == ''">https://dotnetcli.azureedge.net/dotnet/Runtime/$(SharedFrameworkVersion)/</CoreSetupBlobRootUrl>
 
     <CoreSetupBlobAccessTokenParam Condition="'$(DotNetAssetRootAccessTokenSuffix)' != ''">$(DotNetAssetRootAccessTokenSuffix)</CoreSetupBlobAccessTokenParam>
   </PropertyGroup>
 
   <ItemGroup>
     <_DownloadAndExtractItem Include="CombinedSharedHostAndFrameworkArchive">
-      <Url>$(CoreSetupRootUrl)$(SharedFrameworkVersion)/$(CombinedFrameworkHostCompressedFileName)$(CoreSetupBlobAccessTokenParam)</Url>
+      <Url>$(CoreSetupRootUrl)$(CombinedFrameworkHostCompressedFileName)$(CoreSetupBlobAccessTokenParam)</Url>
```

Comment:
Should `CoreSetupRootUrl` be `CoreSetupBlobRootUrl` here? (Or rename `CoreSetupBlobRootUrl`.)

Reply:
Yes.

Reply:
fixed.